pkg/apis/kops/validation/instancegroup.go

```diff
@@ -207,6 +207,12 @@ func CrossValidateInstanceGroup(g *kops.InstanceGroup, cluster *kops.Cluster, st
 		}
 	}
 
+	if g.Spec.EC2InstanceConnect {
```

Comment:
You also only support AWS linux and ubuntu, maybe another check is useful here

Reply:
Checking for distro during validation is a bit tricky. One doesn't really have anything that says anything about the distro.